soccer/gameplay/plays/testing/test_coordinated_pass.py

```diff
@@ -36,9 +36,11 @@ def execute_running(self):
         pass_bhvr = self.subbehavior_with_name('pass')
 
         if pass_bhvr.is_done_running():
-            pass_bhvr.restart()
+            self.remove_subbehavior('pass')
```

Comment:
Any reason for removing, then re-adding the pass behavior instead of just restarting the old one?  The only reason I like `restart()` better here is that it's a good test of whether or not the object state is handled appropriately throughout transitions.